tests/Fixtures/TestBundle/Entity/Container.php

```diff
@@ -33,7 +34,7 @@ class Container
     private $id;
 
     /**
-     * @ApiProperty(subresource=true)
+     * @ApiProperty(subresource=@ApiSubresource())
```

Comment:
I agree, `@ApiSubresource()` directly on the property is easier to type, to read and to understand.